experiments/AMIP/modular/components/atmosphere/climaatmos_init.jl

```diff
@@ -26,5 +27,72 @@ function atmos_init(::Type{FT}, Y, integrator; params = nothing) where {FT}
         @warn("Running with ρe_int in coupled mode is not tested yet.")
     end
 
-    AtmosSimulation(params, Y, spaces, integrator)
+    ClimaAtmosSimulation(params, Y, spaces, integrator)
+end
+
+# required by the Interfacer
+get_field(sim::ClimaAtmosSimulation, ::Val{:radiative_energy_flux}) = level(sim.integrator.p.ᶠradiation_flux, half)
+get_field(sim::ClimaAtmosSimulation, ::Val{:liquid_precipitation}) =
+    sim.integrator.p.col_integrated_rain .+ sim.integrator.p.col_integrated_snow # all fallen snow melts for now
+get_field(sim::ClimaAtmosSimulation, ::Val{:snow_precipitation}) = sim.integrator.p.col_integrated_snow .* FT(0)
+
+get_field(sim::ClimaAtmosSimulation, ::Val{:turbulent_energy_flux}) = sim.integrator.p.ρ_dif_flux_h_tot
+get_field(sim::ClimaAtmosSimulation, ::Val{:turbulent_moisture_flux}) = sim.integrator.p.ρ_dif_flux_q_tot
+
+function update_field!(sim::ClimaAtmosSimulation, ::Val{:surface_temperature}, field)
+    sim.integrator.p.radiation_model.surface_temperature .= RRTMGPI.field2array(field)
+    parent(sim.integrator.p.T_sfc) .= parent(field)
+
+end
+function update_field!(sim::ClimaAtmosSimulation, ::Val{:albedo}, field)
+    sim.integrator.p.radiation_model.diffuse_sw_surface_albedo .=
+        reshape(RRTMGPI.field2array(field), 1, length(parent(field)))
+    sim.integrator.p.radiation_model.direct_sw_surface_albedo .=
+        reshape(RRTMGPI.field2array(field), 1, length(parent(field)))
+end
+function update_field!(sim::ClimaAtmosSimulation, ::Val{:roughness_momentum}, field)
+    parent(sim.integrator.p.sfc_inputs.z0m) .= parent(field)
```

Comment:
Could we rename `z0m` and `z0b` to `roughness_m` and `roughness_b` for clarity?

Reply:
Oh these are variables in `ClimaAtmos`. I'm hoping that we can contain the component model-specific nomenclature in the ...`init_` files only from now on (though we may need to apply these abstractions to some of the other coupler modules before then).